nix/gce.nix

```diff
@@ -193,7 +200,7 @@ let
     options = {
       gce = mkOption {
         default = null;
-        type = with types; uniq (nullOr (submodule gceDiskOptions));
```

Comment:
why remove uniq ?

Reply:
So that we don't find ourselves in a situation where we have to re-define all options 
**Example**
*gce.nix*
```
      fileSystems."/data" = {
        fsType = "xfs";
        options = [ "noatime" "nodiratime" ];
        autoFormat = true;
        formatOptions = "-f";
        gce.disk = null;
        gce.disk_name = null;
        gce.size = lib.mkDefault 500;
        gce.diskType = "ssd";
        gce.encrypt = encrypt;
      };
```
*partitions-gce.nix*
```
 {
   fileSystems."/data".gce.disk      = lib.mkForce resources.gceDisks."${machine}-data";
   fileSystems."/data".gce.disk_name = lib.mkForce "data";
 } 
```
This error is raised 

```
error: The unique option `fileSystems./data.gce' is defined multiple times, in `<unknown-file>' and `/nixops/partitions-gce.nix'.
(use '--show-trace' to show detailed location information)
```
_Solution_

```
      fileSystems."/data".gce = lib.mkOverride 10
      {
        disk      = resources.gceDisks."${machine}-data";
        disk_name = "data";
        size      = volumeSize;
        diskType  = "ssd";
        encrypt   = encrypt;
      };
```